src/stores/spaces/SpaceStore.ts

```diff
@@ -101,6 +101,8 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
     private notificationStateMap = new Map<SpaceKey, SpaceNotificationState>();
     // Map from space key to Set of room IDs that should be shown as part of that space's filter
     private spaceFilteredRooms = new Map<SpaceKey, Set<string>>(); // won't contain MetaSpace.People
+    // Map from space key to Set of room IDs that should be shown as part of that space's filter
+    private spaceFilteredDirectChildRooms = new Map<SpaceKey, Set<string>>(); // won't contain MetaSpace.People
```

Comment:
also presumably does not need to contain the other metaspaces given that'd just be duplicated identical data

Reply:
Not only does not need to contain, simply would not contain, given you only populate it with spaceIds, so the signature needs updating to reflect reality